Js/app.js

```diff
@@ -0,0 +1,14 @@
+const hamburger = document.querySelector('.menu-icon');
```

Comment:
@OLIPLICHE Did you review this project using the live link in the README.md file?

Reply:
> @OLIPLICHE Did you review this project using the live link in the README.md file?

No

Reply:
Thank you for your response! I made some changes and I request you to please check it again and see if the menu appears as my partner has only have 1 review left for this project and it would be helpful if you can check it for us! Thanks